hydpy/models/rconc_uh.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""Calculate runoff concentration by convolution of the isosceles triangular
+unit hydrograph.
+
+|rconc_uh| is a submodel that supplies its main model with the
+calculation of the runoff concentration by convolution of the isoscelese
+triangular unit hydrograph.
+
+The integration test of the application model |hland_v1| uses the
+|rconc_uh| submodel.
+"""
+# import...
+# ...from HydPy
+from hydpy.exe.modelimports import *
+from hydpy.interfaces import rconcinterfaces
+from hydpy.models.rconc import rconc_model
+
+
+class Model(rconc_model.Sub_RConcModel, rconcinterfaces.RConcModel_V1):
+    """Model that calculates runoff concentration by convolution"""
```

Comment:
"."

Also, I am unsure about this sentence's grammar.

Reply:
already changed to "Calculate runoff concentration using the unit hydrograph approach."